irayalina/static/scss/_variable.scss

```
@@ -0,0 +1,13 @@
+$color-primary: rgba(44,83,131,1);
```

Ity tokony `rgba(0,48,73,1);`
```suggestion
$color-primary: rgba(0,48,73,1);
```